Cargo.toml

```diff
@@ -21,7 +21,7 @@ members = [
     "linera-views-derive",
     "linera-witty",
     "linera-witty-macros",
-    "linera-witty/test-modules",
+    "linera-witty/test-modules"
```

Comment:
Nit: just checking if this formatting was on purpose

Reply:
It wasn't, I had the wrong version of `cargo-sort` installed locally :)  Thank you!